lib/loader/index.js

```diff
@@ -14,9 +14,13 @@
  */
 
 var global = require('../global');
+var namespacer = require('../namespacer');
 var doc = global.document;
 
 var DEFAULT_TIMEOUT = 10000;
+var SHARED_NAMESPACE_NAME = 'BV_LOADER_TRACKING';
```

Comment:
I was thinking you'd take an optional namespace as a parameter / setting / something, so that only apps sharing a namespace share this.

For CSS it doesn't matter, but for JS we have the interesting situation in which app in namespace A and app in namespace B may both be using the static asset service to load the same content into different encapsulations in different namespaces.

This approach could mess that up. Thoughts from the other folk?